### PR TITLE
add undo

### LINKED ISSUE #135: New icon animation: undo
**Name of icon to be animated as shown in [Carbon Icon Library](https://carbondesignsystem.com/guidelines/icons/library/):**
undo

**Category of icon:**
operations

**Team/product requesting:**
icon motion group

**Contacts:**
@kristastarr @johnbister 

**When is this needed?**
* optional - is there a specific deadline

**Additional context**
* optional - is there anything else we should know?

### PR DESCRIPTION
**Pull request by johnbister**
this closes #135

### CODE REVIEW

**Comment by johnbister** [comment]
@kristastarr @tay1orjones No rush, but some of the netlify checks are missing and there is no deploy link. Any idea what's happening?

**Comment by kristastarr** [comment]
@johnbister I'm not sure! I am going to log into Netlify and see if I can restart the builds or tell what's wrong

File: src/components/Operations/index.js [hunk 1]
```diff
@@ -100,5 +101,6 @@ export {
   TagExportMotion,
   TagImportMotion,
   TagGroupMotion,
-  TagNoneMotion
+  TagNoneMotion,
+  UndoMotion
```
[the remark this answers is not in the document]
Comment (kristastarr) [reply src/components/Operations/index.js:105]: Thanks! I did not know that! 😄  Should I fix these ones or just do it in the future?

[the remark this answers is not in the document]
Comment (kristastarr) [reply src/components/Operations/index.js:105]: Ok! I'll go ahead and merge then!

**Review by kristastarr — APPROVED** [review]
LGTM! 👍

**Pull request merged**